static/js/l10n/README.md

```diff
@@ -0,0 +1,15 @@
+# FormatJS i18n instructions
```

Comment:
can we move this readme up to root (and maybe renamed to localization.md)

Reply:
we'll have a few of these eventually, so i was thinking it could be all centralized. we can keep this for now and merge later tho